Exiled.API/Enums/EffectType.cs

```diff
@@ -199,5 +199,25 @@ public enum EffectType
         /// Teleports the player to the pocket dimension and drains health until the player escapes or is killed.
         /// </summary>
         PocketCorroding,
+
+        /// <summary>
+        /// Makes walking sound volume equal 0.1.
```

Comment:
reduce by 10% the sound volume of walking (this effect is multiply by the intensity)